packages/stark-ui/src/modules/dropdown/components/dropdown.component.ts

```diff
@@ -0,0 +1,199 @@
+import { Component, EventEmitter, Inject, Input, OnChanges, OnInit, Output, SimpleChanges } from "@angular/core";
+import { TranslateService } from "@ngx-translate/core";
+import { STARK_LOGGING_SERVICE, StarkLoggingService } from "@nationalbankbelgium/stark-core";
+
+/**
+ * Component to display dropdown list based on the options passed as parameters. The dropdown component is based
+ * on the Angular Material MatSelect.
+ */
+@Component({
+	selector: "stark-dropdown",
+	templateUrl: "./dropdown.component.html"
+})
+export class StarkDropdownComponent implements OnInit, OnChanges {
+
+	/**
+	 * (optional) if the dropdown will contain a default blank option
+	 */
+	@Input()
+	public defaultBlank?: boolean;
+	
+	/**
+	 * HTML "id" attribute of the element.
+	 */
+	@Input()
+	public dropdownId: string;
+	
+	/**
+	 * HTML "name" attribute of the element.
+	 */
+	@Input()
+	public dropdownName: string;
+	
+	/**
+	 * header Text to be displayed as the dropdown's header. Such header will be shown at the top of the
+	 * dropdown options list(dynamically translated via the $translate service if the provided text is defined in the translation keys).
+	 */
+	// TODO reenable once a solution for the replacement of md-select-header as been found
+	// @Input()
+	// public header?: string;
+	/**
+	 * (optional) Whether the dropdown is disabled
```

Comment:
In other modules (stark-core), the `(optional)` is at the end of the description. Could you please do the same here ?